### PR TITLE
Add realistic_env_load_sweep forge test, to check latency across TPS

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

File: .github/workflows/forge-stable.yaml [hunk 1]
```diff
@@ -284,18 +284,31 @@ jobs:
...
     if: ${{ github.event_name != 'pull_request' }}
     needs: determine-test-metadata
     uses: aptos-labs/aptos-core/.github/workflows/workflow-run-forge.yaml@main
     secrets: inherit
     with:
       IMAGE_TAG: ${{ needs.determine-test-metadata.outputs.IMAGE_TAG }}
-      FORGE_NAMESPACE: forge-land-blocking-three-region-${{ needs.determine-test-metadata.outputs.IMAGE_TAG }}
-      FORGE_RUNNER_DURATION_SECS: 1800
-      FORGE_TEST_SUITE: land_blocking_three_region
+      FORGE_NAMESPACE: forge-land-blocking-new-${{ needs.determine-test-metadata.outputs.IMAGE_TAG }}
```
Comment (perryjrandall) [inline .github/workflows/forge-stable.yaml:296]: nit, change the namespace to match test name?

File: .github/workflows/forge-stable.yaml [hunk 2]
```diff
@@ -284,18 +284,31 @@ jobs:
...
+      POST_TO_SLACK: true
+
+  run-forge-realistic-env-load-sweep:
+    if: ${{ github.event_name != 'pull_request' }}
+    needs: determine-test-metadata
+    uses: aptos-labs/aptos-core/.github/workflows/workflow-run-forge.yaml@main
+    secrets: inherit
+    with:
+      IMAGE_TAG: ${{ needs.determine-test-metadata.outputs.IMAGE_TAG }}
+      FORGE_NAMESPACE: forge-land-blocking-new-${{ needs.determine-test-metadata.outputs.IMAGE_TAG }}
```
Comment (perryjrandall) [inline .github/workflows/forge-stable.yaml:308]: same

**Review by perryjrandall — APPROVED** [review]

File: testsuite/forge-cli/src/main.rs [hunk 3]
```diff
@@ -228,9 +228,15 @@ fn main() -> Result<()> {
     logger.build();
 
     let args = Args::from_args();
-    let duration = Duration::from_secs(args.duration_secs as u64);
+    let duration = Duration::from_secs(5 * 300 as u64); // args.duration_secs as u64);
```
Comment (perryjrandall) [inline testsuite/forge-cli/src/main.rs:231]: uncomment?

File: testsuite/forge-cli/src/main.rs [hunk 4]
```diff
@@ -228,9 +228,15 @@ fn main() -> Result<()> {
...
     let suite_name: &str = args.suite.as_ref();
 
+    let suite_name = if suite_name == "land_blocking" {
+        "realistic_env_load_sweep"
+    } else {
+        suite_name
+    };
+
```
Comment (ibalajiarun) [inline testsuite/forge-cli/src/main.rs:239]:
```suggestion
```

**Review by ibalajiarun — APPROVED** [review]

**Pull request merged**